.github/workflows/pep8-linting.yml

```diff
@@ -21,6 +21,6 @@ jobs:
       run: |
         pip install flake8
         # stop the build if there are Python syntax errors or undefined names
-        flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
-        # Check for all of the 
-        flake8 . --count --max-complexity=10 --max-line-length=80 --statistics
+        flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics --exclude migrations
```

Comment:
A comment above about why excluding migrations would be useful.
It would also probably be better to do a full path rather than just the folder name.

Reply:
👍 fixed in 83ecdc3.